lerna.json

```diff
@@ -2,5 +2,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "2.1.0"
+  "version": "2.1.1"
```

Comment:
I wouldn't include the version bumps here. I think @epicfaace will handle that once enough changes have amassed, but I'll let him correct me if I'm wrong there.

Reply:
Sorry about that, I accidentally left some changes in when committing after trying the various scripts.

Reply:
Looks like some of the package file changes are still in there.

Reply:
Ok - I found locally and reverted the change, also did the name removal as requested from TextWidget.